docs/commercial/custom_domains.rst

```diff
@@ -0,0 +1,36 @@
+Custom Domains
+==============
+
+Subdomain support
+-----------------
+
+Once a project is imported under Read the Docs,
+by default it's hosted under a subdomain on one of our domains.
+If you need a custom domain, continue on custom domain setup.
+
+Custom domains
+--------------
+
+Projects can also be hosted under a custom domain.
+If you'd prefer to use your own domain name instead of our default hosting domain,
+you can still host with us.
+
+
+We require two steps from your side:
+
+* Add a CNAME record in your DNS that points to our servers ``<organization-slug>.users.readthedocs.com``
+* Add a Domain in the **Project Admin > Domains** page for your project.
```

Comment:
@agjohnson It seems this is not possible to do this by them at this point, since only the "Contact us" button appear in that page.

Reply:
You are right. If you change your project to public, it's possible to add the domain. I'm adding another step for this before this bullet and merging after that.